exe/deploy-complexity.rb

```diff
@@ -167,6 +38,8 @@ def deploy(base, to, options)
           "Github project url to construct links from") do |url|
     options[:gh_url] = url
   end
+  opts.on("--slack-format",
+          "Format output for Slack") { options[:slack_format] = true }
```

Comment:
Oh ... also just realized this output format is not a full blown json file to curl at slack but intermediate format to run eval on for notifier. *THAT* isn't so safe, so I would advise against that part. Either this library should interface with notifier, or it should attempt to output raw json to be pushed at curl, I'm not comfortable with it using eval (though that is a clever hack!)

Reply:
Whoops! I actually intended on adding a "TODO: consult with Charles on whether we can just call Ruby modules directly from this Gem rather than spitting a weird formatted JSON blob out" to this PR, but forgot! I was initially using `eval` during testing, but did not intend to leave this for the final product

Would love to pair on strategies for slack vs. CLI output at some point